src/models/Filter.js

```diff
@@ -1,35 +0,0 @@
-'use strict'
```

Comment:
Did Filter go away as a class because we're storing them in Redux now, so they need to be plain objects?